content/_data/upgrades/2-375-1.adoc

```diff
@@ -1,15 +1,20 @@
-==== Winstone 6.4 and Jetty 10.0.12
+==== Winstone 6.6 and Jetty 10.0.12
 
 Since Jenkins 2.361.1, there have been several Winstone and Jetty updates.
-Jenkins has updated its Winstone version all the way from 6.0 to 6.4, and the Jetty version has been updated from 9.4.46.v20220331 to 10.0.12.
+Jenkins has updated its Winstone version all the way from 6.1 to 6.6, and the Jetty version has been updated from 9.4.46.v20220331 to 10.0.12.
 
 The Winstone and Jetty updates include upgrades that users should perform.
 
 When upgrading Winstone to 6.1 and Jetty to 10.0.11, some flags have been modified.
 Support for OpenSSL-style PEM-encoded RSA private keys has been removed when running Jenkins with the embedded Jetty (Winstone) container and TLS.
+
 Specifically, the `--httpsPrivateKey` and `--httpsCertificate` flags have been removed in favor of the `--httpsKeyStore` flag.
-The removed flags have printed deprecation warnings since 2016 and were implemented with non-standard APIs that have since been removed from Java 17.
-The recommendation is to migrate to the `--httpsKeyStore` option, which takes a keystore as described in the documentation.
+The removed flags have printed deprecation warnings since 2016 and were implemented with non-standard APIs that have since been removed from Java 17. +
+The recommendation is to migrate to the `--httpsKeyStore` option, which takes a keystore as described in the documentation. +
 As of link:https://github.com/jenkinsci/jep/blob/master/jep/229/README.adoc[JEP 229], PKCS12 is the recommended keystore type.
 
-The Winstone 6.4 and Jetty 10.0.12 update makes it so that a previously added `--extraLibFolder` option for use with HTTP/2, can now be removed.
\ No newline at end of file
+Additionally, the `--toolsJar` and `--useJasper` flags have been removed, because they no longer serve a purpose with Java 11 or newer. +
+`--ajp13Port` and `--ajp13ListenAddress` have been removed, they are obsolote since Jetty 9, release 6 years ago. +
```

Comment:
```suggestion
`--ajp13Port` and `--ajp13ListenAddress` have been removed, they are obsolote since Jetty 9, which has been released 6 years ago. +
```